contrib/ansible/.gitignore

```diff
@@ -1,2 +1,4 @@
 myinventory
+all.yml
```

Comment:
since the changes to all.yml showed back up, i'm going to assume this didn't work   :)